Services/CO.CDP.Organisation.WebApi.Tests/UseCase/GetReviewsUseCaseTest.cs

```diff
@@ -0,0 +1,96 @@
+using Moq;
+using FluentAssertions;
+using AutoMapper;
+using CO.CDP.Organisation.WebApi.UseCase;
+using CO.CDP.Organisation.WebApi.Model;
+using CO.CDP.OrganisationInformation.Persistence;
+
+namespace CO.CDP.Organisation.WebApi.Tests.UseCase;
+
+public class GetReviewsUseCaseTests
+{
+    private readonly Mock<IOrganisationRepository> _organisationRepositoryMock;
+    private readonly Mock<IMapper> _mapperMock;
```

Comment:
Our approach was not to mock automapper, but use it directly. This way this test becomes a test for the use case as well as mapping (automapper becomes implementation detail). Otherwise, we'd need to write separate tests for mapping. It's a compromise. 

Here's an example:

https://github.com/cabinetoffice/GCGS-Central-Digital-Platform/blob/ba7fb2ac9249cf434bf9f02c2362148cabfff766/Services/CO.CDP.Organisation.WebApi.Tests/UseCase/GetOrganisationUseCaseTest.cs#L11

Reply:
Done